Sources/FunctionalCss/GridHelpersUpgrade.swift

```diff
@@ -3,33 +3,29 @@ import HtmlUpgrade
 
 // TODO: extract to grid helpers in design systems?
 extension Node {
-  public static func _gridRow(_ attribs: [Attribute<Tag.Div>], _ content: [Node]) -> Node {
-    return .div(attributes: _addClasses([Class.grid.row], to: attribs), .fragment(content))
+  public static func gridRow(attributes: [Attribute<Tag.Div>] = [], _ content: Node...) -> Node {
```

Comment:
Updated the function to match the library a bit more by removing the overload method and give attributes a default.